types/color-support/color-support-tests.ts

```diff
@@ -0,0 +1,48 @@
+import * as colorSupport from 'color-support';
```

Comment:
`import *` is always used to import modules and are not new-able or callable. Use `import colorSupport = require('color-support')` instead.

See this link for a more detailed answer: [typescript import as vs import require](https://stackoverflow.com/questions/35706164/typescript-import-as-vs-import-require).